src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -192,8 +173,8 @@ export default {
 			isCirclesEnabled: loadState('spreed', 'circles_enabled'),
 			canStartConversations: loadState('spreed', 'start_conversations'),
 			initialisedConversations: false,
-			cancelSearchPossibleConversations: () => {},
-			cancelSearchListedConversations: () => {},
+			cancelSearchPossibleConversations: () => { },
```

Comment:
After setting up linter, please look through the file and revert all changes, which are unintended, and not related to this issue

Reply:
Corrected